src/main/java/games/strategy/engine/lobby/server/db/AccessLogDao.java

```diff
@@ -0,0 +1,22 @@
+package games.strategy.engine.lobby.server.db;
+
+import java.sql.SQLException;
+import java.time.Instant;
+
+import games.strategy.engine.lobby.server.User;
+
+/**
+ * Data access object for the access log table.
+ */
+public interface AccessLogDao {
+  /**
+   * Inserts a new record in the access log table.
+   *
+   * @param instant The instant of the access.
+   * @param user The user who accessed the lobby.
+   * @param authenticated {@code true} if the access was authenticated; otherwise {@code false}.
+   *
+   * @throws SQLException If an error occurs while logging the access.
+   */
+  void insert(Instant instant, User user, boolean authenticated) throws SQLException;
```

Comment:
I'd sugggest passing enum value instead of `boolean`, I see the caller does the comparison, we could have DAO take care of that for us and accept the enum value directly. Specifically, suggestion is to replace the type of the 3rd parameter to enum instead of boolean

Reply:
That's what I originally did, but then I felt that might be leaking business logic into the DAO and switched to the `boolean` parameter.  Based on the recent discussion, I agree that passing the `enum` is best (again :smile:).